docs/source/routing/observability/telemetry.mdx

```diff
@@ -20,139 +31,300 @@ flowchart LR
+This configuration snippet enables
+- setting attributes that Datadog uses to organize its APM view
+- exporting traces to a Datadog agent:
+
+```yaml title="router.yaml"
+telemetry:
+  instrumentation:
+    spans:
+      mode: spec_compliant
+      router:
+        attributes:
+          otel.name: router
+          operation.name: "router"
+          resource.name:
+            request_method: true
+      supergraph:
+        attributes:
+          otel.name: supergraph
+          operation.name: "supergraph"
+          resource.name:
+            operation_name: string
+      subgraph:
+        attributes:
+          otel.name: subgraph
+          operation.name: "subgraph"
+          resource.name:
+            subgraph_operation_name: string
+  exporters:
+    tracing:
+      otlp:
+        enabled: true
+        endpoint: "${env.DATADOG_AGENT_HOST}:4317"
+```
 
-## Router telemetry attributes
+Learn more about [sending traces to DataDog](/graphos/reference/router/telemetry/trace-exporters/datadog) and [trace exporters](/graphos/reference/router/telemetry/trace-exporters/overview) in general.
 
-You can annotate events, metrics, and spans with **attributes**. Attributes are key-value pairs that add contextual information about the router pipeline to telemetry. You can then use these attributes to filter and group data in your APMs.
+## Best practices
 
-Example attributes include:
+### Collect exactly the telemetry you need
 
-* HTTP status code
-* GraphQL operation name
-* Subgraph name
+Effective telemetry provides just the right amount and granularity of information to maintain your graph. Too much data can overwhelm your system, for example, with high cardinality metrics. Too little may not provide enough information to debug issues.
 
-You can use [standard attributes](/router/configuration/telemetry/instrumentation/standard-attributes) or [selectors](/router/configuration/telemetry/instrumentation/selectors) as span attributes.
+Specific events that need to be captured—and the conditions under which they need to be captured—can change as client applications and graphs change. Different environments, such as production and development, can have different observability requirements.
 
-<PremiumFeature>
+Router telemetry is customizable to meet the observability needs of different graphs. Keep in mind your particular environments' and graphs' requirements when configuring your telemetry.
 
-[Custom attributes for spans](/router/configuration/telemetry/instrumentation/spans/#attributes) require a GraphOS [Dedicated or Enterprise plan](https://www.apollographql.com/pricing#observability).
+#### Conditions
 
-</PremiumFeature>
+You can set conditions for instruments and events to only collect telemetry data when necessary. This configuration snippet enables only collecting the configured telemetry data when the `request_header` is equal to "example-value":
 
-## Best practices
```

Comment:
Comment from @devfreddy-apollo on this section:

> Yes! This is an area we should expand on for sure. The router team has opinions, CS should have opinions as well. I have some I've started to aggregate but we should crowd source this too. IMO this is where customers get value from all of these great O11y features, and if we don't tell them or do it by default they aren't going to do it.

I agree that we should add more best practices and recipes but would like to get this out-the-door for now. As we expand about best practices, we can move this subsection to its own page, and perhaps eventually to its own set of pages.